src/app.css

```diff
@@ -1,11 +1,4 @@
-.App {
-    min-height: 90vh;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    text-align: center;
-}
-
-h1 {
-    font-size: 5rem;
+body {
+  margin: 0 !important;
```

Comment:
Cześć, nie dodawałbym !important, co prawda w tym przypadku nie powinno to nic zmienić ale trzymajmy się tej zasady, będzie bezpieczniej :)